TIGLViewer/src/TIGLViewerWidget.cpp

```diff
@@ -1267,12 +1267,35 @@ bool TIGLViewerWidget::makeScreenshot(const QString& filename, bool whiteBGEnabl
           .GetRGB()
 #endif
           ;
-        QColor qcol(aColor.Red()*255., aColor.Green()*255, aColor.Blue()*255);
+        QColor qcol(static_cast<int>(aColor.Red()*255),
+                static_cast<int>(aColor.Green()*255),
```

Comment:
As we have no style checks running and since I am really picky 😛 : 

The arguments must be aligned under each other, so all static_casts must be aligned at the same column.